FirebaseRemoteConfig/Tests/FakeConsole/FakeConsoleTests.swift

```diff
@@ -59,6 +60,43 @@ class FakeConsoleTests: APITestBase {
     waitForExpectations()
   }
 
+  // Test New API.
+  // Contrast with testUnchangedActivateWillFlag in APITests.swift.
+  func testChangedActivateWillNotFlag() {
+    let expectation = self.expectation(description: #function)
+    config.fetch { status, error in
+      if let error = error {
+        XCTFail("Fetch Error \(error)")
+      }
+      XCTAssertEqual(status, RemoteConfigFetchStatus.success)
+      self.config.activate { changed, error in
+        XCTAssertNil(error)
+        XCTAssert(changed)
```

Comment:
Tiny nit: Can you change this to `XCTAssertTrue`? I did a double take on it, I think explicitly saying `True` makes it more clear.

Reply:
Agreed! Done